Swift/KVSiOSApp/AppDelegate.swift

```diff
@@ -15,6 +15,22 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
         // Override point for customization after application launch.
 
+        // Do not show Cognito sign-in screen and go directly to channel configuration screen
+        // when not using Cognito integration
```

Comment:
//application uses the  user provided test IAM credentials through environment  (testing only)